desktop/core/src/desktop/js/apps/storageBrowser/StorageFilePage/StorageFilePage.test.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import StorageFilePage from './StorageFilePage';
+import { PathAndFileData } from '../../../reactComponents/FileChooser/types';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+
+// Mock the translation function
+jest.mock('../../../utils/i18nReact', () => ({
+  i18nReact: {
+    useTranslation: () => ({
+      t: (key: string) => key
+    })
+  }
+}));
+
+// Mock data for fileData
+const mockFileData: PathAndFileData = {
+  path: '/path/to/file.txt',
+  stats: {
+    size: 123456,
+    user: 'testuser',
+    group: 'testgroup',
+    mtime: '1617877200',
+    atime: '1617877200',
+    blockSize: 4096,
+    mode: 33188,
+    replication: 1,
+    path: '/path/to/file.txt'
+  },
+  rwx: 'rwxr-xr-x',
+  breadcrumbs: [],
+  view: {
+    contents: 'Initial file content'
+  },
+  files: [],
+  page: {
+    number: 1,
+    num_pages: 1,
+    previous_page_number: 1,
+    next_page_number: 1,
+    start_index: 1,
+    end_index: 1,
+    total_count: 1
+  },
+  pagesize: 100
+};
+
+describe('StorageFilePage', () => {
+  it('renders file metadata and content', () => {
+    render(<StorageFilePage fileData={mockFileData} />);
+
+    expect(screen.getByText('Size')).toBeInTheDocument();
+    expect(screen.getByText('120.56 KB')).toBeInTheDocument();
+    expect(screen.getByText('Created By')).toBeInTheDocument();
+    expect(screen.getByText('testuser')).toBeInTheDocument();
+    expect(screen.getByText('Group')).toBeInTheDocument();
+    expect(screen.getByText('testgroup')).toBeInTheDocument();
+    expect(screen.getByText('Permissions')).toBeInTheDocument();
+    expect(screen.getByText('rwxr-xr-x')).toBeInTheDocument();
+    expect(screen.getByText('Last Modified')).toBeInTheDocument();
+    expect(screen.getByText('April 8, 2021 at 03:50 PM')).toBeInTheDocument();
```

Comment:
I think this is failing in your PR tests due to the differences in time zones between your dev env and the CI environment.